services/web/server/src/simcore_service_webserver/version_control_db.py

```diff
@@ -40,6 +42,10 @@
 
 
 def compute_checksum(workbench: Dict[str, Any]) -> SHA1Str:
+    #
+    # - UI is NOT accounted in the checksum
+    # - TODO: review other fields to mask?
+    #
```

Comment:
Could you clear out this a bit more?

Reply:
Aha sorry, this got here while it was a comment intended for PR #2556. I will remove it from here.